### PR TITLE
chore: modify example to execute in IE

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* modify example code to execute in IE
* What is the effective way to maintain example code?
  * using [babel-standalone](https://babeljs.io/docs/en/babel-standalone) **vs** seperated code(real code to execute, displaying code in custom tag)

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: packages/toast-ui.grid/examples/example03-custom-editor.html [hunk 1]
```diff
@@ -126,10 +259,10 @@
 			{
 				header: 'Grade',
 				name: 'grade',
-				onBeforeChange: function(ev){
+				onBeforeChangeev) {
```
Comment (jung-han) [inline packages/toast-ui.grid/examples/example03-custom-editor.html]: 오타인듯 합니다

File: packages/toast-ui.grid/examples/example04-custom-renderer.html [hunk 2]
```diff
@@ -14,10 +14,62 @@
...
+			this.render(props);
+		}
+
+		CustomSliderRenderer.prototype.getElement = function() {
+			return this.el;
+		}
+
+		CustomSliderRenderer.prototype.render = function(props) {
+			this.el.value = String(props.value);
+		}
```
Comment (jung-han) [inline packages/toast-ui.grid/examples/example04-custom-renderer.html]: `;`로 닫아줘야 할 것 같습니다. html 예제 페이지 전부 프리티어를 한번 돌리면 좋을 것 같습니다!

File: packages/toast-ui.grid/examples/example07-themes.html [hunk 3]
```diff
@@ -89,144 +89,284 @@ <h4>Custom options</h4>
...
+        tui.Grid.applyTheme(preset);
+      })
+    });
 
-  document.querySelector('.btn-custom').addEventListener('click', () => {
-    const options = eval('(' + document.getElementById('custom-text').value + ')');
-    tui.Grid.applyTheme('default', options);
-  });
+    document.querySelector('.btn-custom').addEventListener('click', () => {
+      const options = eval('(' + document.getElementById('custom-text').value + ')');
```
Comment (jung-han) [inline packages/toast-ui.grid/examples/example07-themes.html]: template literal을 사용해도 좋을 것 같습니다.

File: packages/toast-ui.grid/examples/example03-custom-editor.html (hunk 1 above)
Comment (seonim-ryu) [reply packages/toast-ui.grid/examples/example03-custom-editor.html]: 변환이 잘못된듯요 ㄷㄷ

**Pull request merged**